pymeasure/instruments/srs/sr830.py

```diff
@@ -416,7 +416,7 @@ def output_conversion(self, channel):
         """
         offset, expand = self.get_scaling(channel)
         sensitivity = self.sensitivity
-        return lambda x: (x / (10. * expand) + offset) * sensitivity
+        return lambda x: x / expand + offset / 100. * sensitivity
```

Comment:
You do not need to divide by the float 100. The results (for python 3) are the same with `/ 100`.

Reply:
This is just a note for yourself.

Reply:
I now removed the ".". While I was aware of the python 3 feature i wanted to be explicit here.